app/grandchallenge/hanging_protocols/forms.py

```diff
@@ -160,7 +161,12 @@ def __init__(self, *args, **kwargs):
             reverse("documentation:detail", args=["viewer-content"]),
         )
 
-    def generate_view_content_example(self):
+    def _get_interface_lists(self):
+        interface_lists = namedtuple(
+            "interface_lists",
+            ["images", "mandatory_isolation_interfaces", "overlays"],
+        )

```

Comment:
Is there a reason you used `collections.namedtuple` instead of `typing.NamedTuple` like we do in the majority of places in the codebase?

Reply:
I changed it to `typing.NamedTuple`. It is safer to do it with type checking. 
Done